src/Task/Phpunit.php

```diff
@@ -21,12 +21,22 @@ public static function getConfigurableOptions(): OptionsResolver
             'testsuite' => null,
             'group' => [],
             'always_execute' => false,
+            'order' => 'default',
         ]);
 
         $resolver->addAllowedTypes('config_file', ['null', 'string']);
         $resolver->addAllowedTypes('testsuite', ['null', 'string']);
         $resolver->addAllowedTypes('group', ['array']);
         $resolver->addAllowedTypes('always_execute', ['bool']);
+        $resolver->addAllowedValues('order', [
```

Comment:
I am not sure wether to enforce these values from grumphp. I assume that the list will grow in time and needs to keep up-to date.
If phpunit fails on invalid order, it might make sense to let phpunit do the validation instead of grumphp?
That way we don't have to keep this list in sync with phpunit?

Reply:
done